frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/AttachmentPortletUtils.java

```diff
@@ -127,8 +127,7 @@ private void serveAttachmentBundle(List<AttachmentContent> attachments, Resource
         String filename;
         String contentType;
         if(attachments.size() == 1){
-            filename = downloadFileName
-                    .orElse(attachments.get(0).getFilename());
+            filename = attachments.get(0).getFilename();
```

Comment:
`downloadFileName` is in case of 1 attachment not valid.
Its the name of zip archive like `SourceCodeBundle-projectx-time-.zip`
This misrepresent the original file extension

Reply:
You are right. It looks like that this feature is never used.